components/server/src/server.rs

```diff
@@ -1107,7 +1107,7 @@ impl<ER: RaftEngine> TiKVServer<ER> {
                     .get_engine_size()
                     .expect("get raft engine size");
 
-                let used_size = snap_size + kv_size + raft_size;
+                let used_size = snap_size + kv_size + raft_size + reserve_space / 5;
```

Comment:
Also needs to `fstat` the place holder file.

Reply:
Ok, add stats.